templates/project/CONTRIBUTING.md

```diff
@@ -315,26 +327,26 @@ If the deprecated thing is a service, you **MUST** specify it on the service def
 
 More info: http://symfony.com/blog/new-in-symfony-2-8-deprecated-service-definitions
 
-For everything else, not managed by the `@deprecated` tag, you **MUST** trigger a deprecation message.
+For everything else, not managed by the `@deprecated` tag,
+you **MUST** still trigger a deprecation message (and add a `NEXT_MAJOR` comment).
 
 ```php
 <?php
+// NEXT_MAJOR: Remove this condition.
 if (/* some condition showing the user is using the legacy way */) {
     @trigger_error(
-        'The '.__METHOD__.' method is deprecated since sonata-project/bar-bundle 42.x, to be removed in 43.0. '.
-        'Use FooClass::barMethod() instead.',
+        'This case is deprecated since sonata-project/bar-bundle 42.x and will not be supported in 43.0.'
```

Comment:
Just to be consistent with the previous deprecation message:
```suggestion
        'This case is deprecated since sonata-project/bar-bundle 42.x and will not be supported in version 43.0.'
```